test/func/mocha.opts

```diff
@@ -0,0 +1,3 @@
+test/func/spec/**/*.spec.js
+--recursive
+--timeout 50000
```

Comment:
The long timeout is for running the tests locally against the webpack dev server, which won't start compiling until the first test hits the server. I've reduced it to something a little more reasonable I think, though.